attribute/old/attribute_target.c

```diff
@@ -75,6 +75,7 @@ struct {
 
 	{ "TARGET_TYPE_TPM", "tpm", "tpm" },
 	{ "TARGET_TYPE_BMC", "bmc", "bmc" },
+	{ "TARGET_TYPE_OSCREFCLK", "oscrefclk", "oscrefclk" },
```

Comment:
This is the old attribute code.  The new code uses libdtree.  So you will have to modify `libdtree/dtree_util.c`.

Reply:
Thanks for pointing - I did not notice during rebase.

Added this line in `libdtree` and removed in the old code